.circleci/config.yml

```diff
@@ -1,16 +1,253 @@
-version: 2.1 # Use v2.1 to enable orb usage.
-
-# The Windows orb gives you everything you
-# need to start using the Windows executor.
+version: 2.1
+parameters:
+  buildReleaseType:
+    description: 'Semantic Version increment type of the release. Valid values: major, minor, patch'
+    default: "patch"
+    type: enum
+    enum: ["major", "minor", "patch"]
 orbs:
   win: circleci/windows@2.2.0
+build-and-sign-ADMU: &build-and-sign-ADMU
+  - Build
+  - Pester Tests:
+      requires:
+        - Build
+  # - Sign Executable:
```

Comment:
we cant sign using circle ci yet - this is commented out and we use azure dev ops in the meantime